templates/standard.adoc

```diff
@@ -29,7 +28,8 @@ image::pomegranate.png[Pomegranate]
 * `pos`: `vector`
 * `name`: `string`
 
-Plants a tree of type `name` at `pos` if `name` is a registered pomegranate tree.
+Plants a tree of type `name` at `pos`
```

Comment:
That seems excessive to me

Reply:
Perhaps, but we can change it if a better syntax is found.